run_dir/static/js/projects.js

```diff
@@ -129,13 +140,10 @@ function load_table_head(columns){
 }
 
 
-
-
-// Undefined columns handled here
-function load_undefined_columns() {
-  return $.getJSON("/api/v1/projects_fields?undefined=true&project_list=" + projects_page_type, function(data) {
+// Undefined columns handled here now
+function load_undefined_columns(cols) {
     var columns_html = "";
-    $.each(data, function(column_no, column) {
+    $.each(cols, function(col_id, column) {
       $("#undefined_columns").append('<div class="checkbox">'+
           '<label>'+
             '<input type="checkbox" class="filterCheckbox" data-columngroup="undefined-columns" data-displayname="'+column+'" name="'+column+'" id="undefined-columns-'+column+'">'+
```

Comment:
not using, the `col_id`, right? Just checking that you didn't typo it with `column`